packages/server/tests/acceptance.spec.ts

```diff
@@ -398,14 +400,16 @@ describe('RPC Server Integration Tests', async function () {
         expect(senderInitialBalance).to.not.eq(senderEndBalance);
         expect(utils.subtractBigNumberHexes(senderInitialBalance, senderEndBalance).toHexString()).to.not.eq(oneHbarInWeiHexString);
         expect(BigNumber.from(senderInitialBalance).sub(BigNumber.from(senderEndBalance)).gt(0)).to.eq(true);
+
+        ethCompNonce++;
```

Comment:
I don't think that adding dependency between tests (in this case `ethCompNonce`) is a good approach. I would rather get the `nonce` prior to submitting the transaction.